### ISSUE TITLE
Include contact links in member bio

### ISSUE THREAD

**Issue by alliequintano** [cherryontech/website#112]

**Comment by novellac** [comment]
As part of this, we need icons for:
- LinkedIn
- Medium
- Dribble
- Instagram
- Twitter
- Github
- Generic link to user portfolio

@julietafb Do you want to use FontAwesome across the site? Did we decide on an icon pack? FontAwesome has free icons for each of these.

**Comment by julietafb** [comment]
Yes, let's do that for the icons! I've been using mostly from figma plugin

**Comment by julietafb** [comment]
please review https://zpl.io/aMyWQwN

**Comment by novellac** [comment]
The links are in but we still need icons.
